models/professor_test.py

```diff
@@ -0,0 +1,41 @@
+import rmc.models as m
+import rmc.test.lib as testlib
+
+
+class ProfessorTest(testlib.FixturesTestCase):
+
+    def test_transfer_reviews_from(self):
+        real_prof = m.Professor(
+            id='real_prof'
+        )
+        dupe_prof = m.Professor(
+            id='dupe_prof'
+        )
+
+        DUPE_PROF_REVIEWS_START_NUM = 5
```

Comment:
I'm not sure what DUPE_PROF_REVIEWS_START_NUM means. Would NUM_DUPE_PROF_REVIEWS make more sense?